lib/fathead/spring-framework/fetch_docs.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python
```

Comment:
I don't know how automated your procedures are.  The idea of using the 'html' docs and not the zip was that by using the 'html' processing you always get the docs for the latest stable release, as signified by the 'current' in the url, without human intervention.  I will try to see if I can get it to work similarly with the zip.